### PR TITLE
Test cover 'toImage'.

### PR DESCRIPTION
**Pull request by polina-c**
Pre-work for https://github.com/flutter/engine/pull/35274.

### CODE REVIEW

File: lib/web_ui/test/canvaskit/image_test.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+void testMain() {
+  setUpCanvasKitTest();
+
+  test('toImage succeeds', () async {
+    final ui.Image image = await _createImage();
+    expect(image.runtimeType.toString(), equals('CkImage'));
+    image.dispose();
+  // TODO(hterkelsen): https://github.com/flutter/flutter/issues/109265
+  }, skip: isFirefox || isSafari);
```
Comment (yjbanov) [inline lib/web_ui/test/canvaskit/image_test.dart:26]: Safari should work now. I'm actually working on a separate PR to unskip all the old tests for Safari.

File: lib/web_ui/test/engine/image/image_test.dart [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+Future<void> testMain() async {
+  // TODO(polina-c): uncomment the test when bug is fixed:
+  // https://github.com/flutter/engine/pull/35791
+  // test('toImage succeeds', () async {
+  //   final ui.Image image = await _createImage();
+  //   expect(image.runtimeType.toString(), equals('HtmlImage'));
+  //   image.dispose();
+  // });
```
Comment (yjbanov) [inline lib/web_ui/test/engine/image/image_test.dart:20]: Maybe keep the code uncommented, but `skip` it? This way at least the code can be analyzed.

**Review by yjbanov — APPROVED** [review]

File: lib/web_ui/test/engine/image/image_test.dart (hunk 2 above)
Comment (polina-c) [reply lib/web_ui/test/engine/image/image_test.dart:20]: done

File: lib/web_ui/test/canvaskit/image_test.dart (hunk 1 above)
Comment (polina-c) [reply lib/web_ui/test/canvaskit/image_test.dart:26]: done

**Pull request merged**